libs/coin-polkadot/src/signer/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * This directory is the home for all types and logic based on Ledgers signer.
+ */
+
+import resolver from "./hw-getAddress";
+
+export default resolver;
```

Comment:
Maybe the occasion to change that weird "resolver" name, and replace with explicit import?